athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/ListTablesRequest.java

```diff
@@ -32,23 +32,50 @@
         extends MetadataRequest
 {
     private final String schemaName;
+    private final String nextToken;
+    private final int pageSize;
 
     /**
      * Constructs a new ListTablesRequest object.
      *
      * @param identity The identity of the caller.
      * @param queryId The ID of the query requesting metadata.
      * @param catalogName The catalog name that tables should be listed for.
-     * @param schemaName The schema name that tables should be listed for. This may be null if no specific schema is requested.
+     * @param schemaName The schema name that tables should be listed for. This may be null if no specific schema is
+     *                   requested.
      */
     @JsonCreator
     public ListTablesRequest(@JsonProperty("identity") FederatedIdentity identity,
-            @JsonProperty("queryId") String queryId,
-            @JsonProperty("catalogName") String catalogName,
-            @JsonProperty("schemaName") String schemaName)
+                             @JsonProperty("queryId") String queryId,
+                             @JsonProperty("catalogName") String catalogName,
+                             @JsonProperty("schemaName") String schemaName)
+    {
+        this(identity, queryId, catalogName, schemaName, null, -1);
```

Comment:
Can we change -1 to a variable and make it a more descriptive name? whenever anyone sees this, or any future maintainers, they shoudn't have to hunt down what it means. It can be a simple NULL_SERIALIZED_PAGE_SIZE or something like that with a comment about what it is.

Reply:
This constructor has been removed. Added a static variable to ListTablesRequest that can be used for that purpose.